serving/fn/openapi/SepsisDetection.json

```diff
@@ -0,0 +1,282 @@
+{
+    "openapi": "3.0.2",
+    "info": {
+        "title": "Sepsis Detection",
+        "version": "1.0.0",
+        "description": "Interface for sending patient data to analyze for sepsis risk assessment. The data points below are optional. Send \"NaN\" if data is null.\n\n[\"Age\", \"Unit1\", \"Unit2\", \"HospAdmTime\", \"ICULOS\", \"Gender\", \"Bilirubin_direct\", \"TroponinI\", \"isSepsis\"]"
```

Comment:
the Data cleaning pipeline built in the data-science.ipynb will replace NaN with the median value of the column and apply a standard scaler to all numeric values - so NaN will get replaced in the pipeline and changing the values from string to float64 will allow them to be scaled

Reply:
@dvanbalen this statement about sending NaN can now be removed